Rytis_George_Baltakys/rails/dojo_ninjas/db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+Dojo.create(
+	[
```

Comment:
http://guides.rubyonrails.org/active_record_querying.html#find-or-build-a-new-object is cool